src/cmake/GeosxOptions.cmake

```diff
@@ -52,7 +52,7 @@ option( ENABLE_SUITESPARSE "Enables SUITESPARSE" ON )
 ### LAI SETUP ###
 
 set( supported_LAI Trilinos Hypre Petsc )
-set( GEOSX_LA_INTERFACE "Trilinos" CACHE STRING "Linear algebra interface to use in solvers" )
+set( GEOSX_LA_INTERFACE "Hypre" CACHE STRING "Linear algebra interface to use in solvers" )
```

Comment:
I think this is intended. We want _hypre_ to become the default LAI.

Reply:
Wasn't there another PR that was also doing this?

Reply:
Not anymore, everything is part of this PR now. We discussed this in the last developer meeting.

Reply:
That PR got folded into this one. However, it seemed to me the conclusion of the last meeting was to fix the linear solver for Hydrofracture _before_ we switch the default LAI.

Reply:
@klevzoff I see your point. However, the urgency of merging this PR because of a dependence of the Sphinx documentation branch on this PR. That constraint has been removed now.

@CusiniM when do you plan to work on the monolithic assembly for the Hydrofracture solver?

Reply:
@klevzoff @castelletto1 I will try to have the hydrofracture solver done in the first half of next week. Hopefully by the next dev meeting.